spec/spec_helper.rb

```diff
@@ -5,12 +5,12 @@
 
 CENTOS_6 = {
   platform: 'centos',
-  version: '6.8',
+  version: '6.9',
 }.freeze
 
 CENTOS_7 = {
   platform: 'centos',
-  version: '7.2.1511',
+  version: '7.4.1708',
 }.freeze
 
 UBUNTU_14_04 = {
```

Comment:
@ramereth Do we still need to be supporting Ubuntu 14.04?

Reply:
Yes, and we should technically add support for the newer versions but I don't think that @csrs should do that in this PR.